fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -103,47 +103,36 @@ ACTOR Future<Void> moveKeySelectorOverRangeActor(const SpecialKeyRangeBaseImpl*
 	return Void();
 }
 
-void onModuleRead(ReadYourWritesTransaction* ryw, SpecialKeySpace::MODULE module,
-                  Optional<SpecialKeySpace::MODULE>& lastModuleRead) {
-	if (ryw && !ryw->specialKeySpaceRelaxed() && lastModuleRead.present() && lastModuleRead.get() != module) {
-		throw special_keys_cross_module_read();
-	}
-	lastModuleRead = module;
-}
-
 // This function will normalize the given KeySelector to a standard KeySelector:
 // orEqual == false && offset == 1 (Standard form)
 // If the corresponding key is outside the whole space, it will move to the begin or the end
 // It does have overhead here since we query all keys twice in the worst case.
 // However, moving the KeySelector while handling other parameters like limits makes the code much more complex and hard
 // to maintain; Thus, separate each part to make the code easy to understand and more compact
+// Boundary is the range of the legal key space, which, by default is the range of the module
+// And (\xff\xff, \xff\xff\xff\xff) if SPECIAL_KEY_SPACE_RELAXED is turned on
```

Comment:
I think the end of the special key space is \xff\xff\xff

Reply:
Right. My bad typing here. Sorry for the confusion.